### DISCUSSION TITLE
Adressen met meerdere nummeraanduiding ID's

### DISCUSSION THREAD

**Discussion by siccovansas** [nlextract/NLExtract#374] [Q&A] [answered]
Sommige adressen hebben meerdere nummeraanduiding ID's. Zo heeft Teunisbloemlaan 50, Vleuten twee nummeraanduiding ID's:

- [0344200000113144](https://bagviewer.kadaster.nl/lvbag/bag-viewer/?searchQuery=0344200000113144&objectId=0344200000113144&theme=BRT+Achtergrond&geometry.x=128625.56797001688&geometry.y=456370.08450898767&zoomlevel=10.998619955827534)
- [0344200000208275](https://bagviewer.kadaster.nl/lvbag/bag-viewer/?searchQuery=0344200000208275&objectId=0344200000208275&theme=BRT+Achtergrond&geometry.x=128610.864&geometry.y=456478.159&zoomlevel=14.644472047196913)

In de "[BAG Adressen - Uitgebreid - CSV](https://geotoko.nl/datasets?rid=24b6070d-5a92-4109-8fcd-a1d7cc000801)" is er echter enkel '0344200000208275' terug te vinden.

Ik heb een beetje rond gezocht in deze mailinglist en op GitHub, maar ik zie niet zo snel of dit zo hoort of niet. Mocht dit wel zo horen, waar kan ik dan een compleet overzicht vinden van adressen + nummeraanduiding ID's?

NB: een ander voorbeeld van een adres met meerdere nummeraanduiding ID's is Houtwijklaan 20, 's-Gravenhage.

**Comment by Puccinistraat** [comment]
Hallo, ik probeer uit te vinden waarom bij postcode 3816 VE-22 vaak de Weberstraat 22 in Amersfoort tevoorschijn komt, terwijl het de Puccinistraat (om de hoek) moet zijn. Weberstraat 22 is 3816 VC. In welke database kan deze fout veroorzaakt worden? 
In de BAG verschijn het correcte adres.

**Comment by justb4** [comment, marked as answer]
> Sommige adressen hebben meerdere nummeraanduiding ID's. Zo heeft Teunisbloemlaan 50, Vleuten twee nummeraanduiding ID's:
> 
> * [0344200000113144](https://bagviewer.kadaster.nl/lvbag/bag-viewer/?searchQuery=0344200000113144&objectId=0344200000113144&theme=BRT+Achtergrond&geometry.x=128625.56797001688&geometry.y=456370.08450898767&zoomlevel=10.998619955827534)
> * [0344200000208275](https://bagviewer.kadaster.nl/lvbag/bag-viewer/?searchQuery=0344200000208275&objectId=0344200000208275&theme=BRT+Achtergrond&geometry.x=128610.864&geometry.y=456478.159&zoomlevel=14.644472047196913)
> 
> In de "[BAG Adressen - Uitgebreid - CSV](https://geotoko.nl/datasets?rid=24b6070d-5a92-4109-8fcd-a1d7cc000801)" is er echter enkel '0344200000208275' terug te vinden.
> 
> Ik heb een beetje rond gezocht in deze mailinglist en op GitHub, maar ik zie niet zo snel of dit zo hoort of niet. Mocht dit wel zo horen, waar kan ik dan een compleet overzicht vinden van adressen + nummeraanduiding ID's?
> 
> NB: een ander voorbeeld van een adres met meerdere nummeraanduiding ID's is Houtwijklaan 20, 's-Gravenhage.

Dit betreft twee verschillende Verblijfsobjecten met verschillende statussen:

* [0344010000101403](https://bagviewer.kadaster.nl/lvbag/bag-viewer/?searchQuery=0344010000101403&objectId=0344010000101403&theme=BRT+Achtergrond&geometry.x=128610.864&geometry.y=456478.159&zoomlevel=14.21963692085059&bijbehorendeAdressen=Pand) - "Verblijfsobject in gebruik"
* [0344010000205103](https://bagviewer.kadaster.nl/lvbag/bag-viewer/?searchQuery=0344010000205103&objectId=0344010000205103&theme=BRT+Achtergrond&geometry.x=128610.864&geometry.y=456478.159&zoomlevel=14.21963692085059&bijbehorendeAdressen=Pand)  - "Verblijfsobject gevormd"

Alleen 0344010000101403 is geldig, 0344010000205103 bijv in aanbouw. Daarom alleen bijbehorende nummeraanduiding 0344200000208275 in adressen CSVs.

**Reply by siccovansas** [reply]
Hi @justb4,

Dank voor de uitleg! Er lijkt toch nog iets niet te kloppen dan.

Het Nummeraanduiding ID [0344200000208275](https://bagviewer.kadaster.nl/lvbag/bag-viewer/?searchQuery=0344200000208275&objectId=0344200000208275&theme=BRT+Achtergrond&geometry.x=128610.864&geometry.y=456478.159&zoomlevel=14.644472047196913) hoort bij Verblijfsobject ID [0344010000205106](https://bagviewer.kadaster.nl/lvbag/bag-viewer/?searchQuery=0344010000205106&objectId=0344010000205106&theme=BRT+Achtergrond&geometry.x=128610.864&geometry.y=456478.159&zoomlevel=14.693076476858977) en die heeft juist 'Verblijfsobject gevormd' (dus niet geldig) en zou dus niet in de adressen CSVs moeten zitten?

Juist Nummeraanduiding ID [0344200000113144](https://bagviewer.kadaster.nl/lvbag/bag-viewer/?searchQuery=0344200000113144&objectId=0344200000113144&theme=BRT+Achtergrond&geometry.x=128625.56797001688&geometry.y=456370.08450898767&zoomlevel=10.998619955827534) zou wel in de adressen CSVs moeten zitten, want het bijbehorende Verblijfsobject ID heeft de geldigde status 'Verblijfsobject in gebruik'.

**Reply by justb4** [reply]
Ik heb op een of andere manier gekeken naar Teunisbloemlaan 44, Vleuten, ipv. nr 50, dus mijn betoog boven klopt niet voor nr 50. Wel is er op nr 44 eenzelfde situatie. 

Maar er is mogelijk wel degelijk een probleem. Zie ook https://github.com/nlextract/NLExtract/issues/173 . ''Verblijfsobject gevormd'' geldt sindsdien als 'bestaand', dus zit in 'verblijfsobjectactueelbestaand' VIEW. We proberen dubbele adressen te vermijden, dus mogelijk valt er een willekeurig af. Normaal gesproken verwacht je aansluiting op begin/einddatum.

Heeft dieper onderzoek nodig...

**Reply by siccovansas** [reply]
Hi @justb4,
Er komen nieuwe verkiezingen aan, dus we bestellen binnenkort weer een nieuwe versie van "[BAG Adressen - Uitgebreid - CSV](https://geotoko.nl/datasets?rid=24b6070d-5a92-4109-8fcd-a1d7cc000801)". Als ik de commits bekijk dan lijkt deze issue nog niet verholpen. Klopt dat? Zo ja, is er zicht op wanneer het opgelost kan worden?

**Reply by justb4** [reply]
Ik kan het op dit moment niet zeggen. Het blijven altijd een paar gevallen, soms ook door fouten in BAG zelf, bijv ontbreken einddatum, maar dan wordt ID met recentste begindatum genomen. Als je een aantal voorbeelden kunt geven van de huidige adressenbestanden van geotoko, kan ik ermee aan de slag.

**Reply by justb4** [reply]
@siccovansas we werken nu aan oplossing via issue #409 .

**Reply by siccovansas** [reply]
Super! We zijn de afgelopen weken weer druk aan de slag geweest met WaarIsMijnStemlokaal.nl en liepen weer een aantal keer tegen dit probleem aan (d.w.z. gemeenten die een BAG Nummeraanduiding ID voor een stembureau bij ons aanleveren, maar die wij niet in 'BAG Adressen - Uitgebreid - CSV' terugvinden).

Voor de aanstaande Tweede Kamerverkiezingen komt een fix voor ons te laat, maar met alweer gemeenteraadsverkiezingen in maart 2026 zou het geweldig zijn als dit opgelost wordt :heart:!

**Reply by justb4** [reply]
De oplossing voor #409 is in ieder geval gedaan en komt al mee in de geotoko BAG van 8 okt 2025.

**Reply by siccovansas** [reply]
Geweldig, dank!

Ik zie dat er dus eerst willekeurig werd gekozen tussen de BAG Nummeraanduiding ID's als er twee beschikbaar waren voor een adres. Na deze oplossing wordt er dus op een consistente manier voor een ID gekozen op basis van status. Er blijft dus maar 1 BAG Nummeraanduiding ID beschikbaar in de 'BAG Adressen - Uitgebreid - CSV'? Wellicht dat dit ons probleem oplost. Maar ik kan me voorstellen dat sommige gemeenten soms ook het andere ID gebruiken. We gaan het zien bij de volgende verkiezingen :). Beide ID's opnemen zou voor ons beter zijn, maar dat is voor anderen misschien niet wenselijk?

**Reply by justb4** [reply]
Ja klopt bij ontdubbelen wordt steeds een 'bestaande' status: `'Verblijfsobject in gebruik (niet ingemeten)', 'Verblijfsobject in gebruik',' Verbouwing verblijfsobject'` gekozen boven 'niet-bestaand' zoals `'Verblijfsobject gevormd'`. 
Er zijn meer ontdubbelslagen, maar ook fouten in de BAG. Als we dubbele adressen zouden opnemen, zouden er meer mensen/toepassingen 'aangedaan' zijn dan omgekeerd. Het ooit meenemen van `'Verblijfsobject gevormd'`, in feite 'niet-bestaand', was al een randgeval, die uiteindelijk ook voor de dubbeling zorgde.

Maar de stembureau's zouden er toch in moeten zitten, als het geldige adressen zijn. Anders een BAG terugmelding doen. Of hier als het echt een fout in NLExtract lijkt.
Je zou eens kunnen buurten bij de GeoDienst van de RUG, die gebruiken die stembureau-adressen ook, bijv [stemwegwijzer.nl](https://stemwegwijzer.nl/) en zie [geodienst.xyz/verkiezingen](https://geodienst.xyz/verkiezingen/). Edit: best oud 2018, zie ik nu...

**Reply by justb4** [reply]
De laatste BAG (8 okt 2025) is net in geotoko gedraaid en vrijgegeven, en heeft ook het oorspronkelijke probleem hierboven opgelost (naast de vele andere cases): VBO `0344010000205106` heeft nog steeds de status `'Verblijfsobject gevormd'`, maar zit nu niet in adres-bestand. Maar wel dus de gewenste VBO `0344010000101405` voor "Teunisbloemlaan 50, Vleuten", [Jenaplanschool, De Weide Wereld](https://bagviewer.kadaster.nl/lvbag/bag-viewer/?searchQuery=0344200000113144&objectId=0344200000113144&theme=Luchtfoto%27s&geometry.x=128608.92220118713&geometry.y=456476.84354987607&zoomlevel=13.346928849765812).

**Reply by siccovansas** [reply]
Super, dank @justb4 :pray:! Voor de gemeenteraadsverkiezingen zullen we weer een nieuwe versie van jullie BAG CSV inladen. Hopelijk lost dit het probleem dus helemaal op. Mochten er dan toch nog andere BAG Nummeraanduiding ID's aangeleverd worden door gemeenten, dan kan ik in ieder geval aan hen laten weten op welke manier de BAG Nummeraanduiding ID's bij ons gekozen zijn. Wellicht dat zij deze manier dan ook kunnen gaan aanhouden.
